schedule/alp/alp_yast_as_workload.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: alp_yast_as_workload
+description: >
+    Enable YaST automation test for ALP
```

Comment:
doesn't reflect that we "run YaST as a workload for ALP"

Reply:
Done